### PR TITLE
feat: Reload website on backend switch

### PR DESCRIPTION
**Pull request by stormofice**

### CODE REVIEW

**Comment by maanex** [comment]
Please put the reload function inside the if statement above. If the popup is dismissed by the user the value is null. We don't need to reload if the user dismisses.

**Comment by stormofice** [comment]
oops fixed

File: composables/view.ts
Comment (maanex) [inline composables/view.ts:1]: why is this getting deleted now? 😳

Comment (stormofice) [reply composables/view.ts:1]: abcdefixedghijklmnopqrstuvwxyz

**Review by maanex — APPROVED** [review]

**Pull request merged**